WpNginxConfig.conf

```diff
@@ -0,0 +1,44 @@
+server {
+    listen 443 ssl;
+    listen [::]:443 ssl;
+    include snippets/self-signed.conf;
+    include snippets/ssl-params.conf;
+
+
+    server_name 192.168.50.2;
+    root /var/www/wordpress;
+
+    index index.php index.html index.htm;
+
+    location / {
+        #try_files $uri $uri/ =404;
+         try_files $uri $uri/ /index.php$is_args$args;
+    }
+
+    location ~ \.php$ {
+        include snippets/fastcgi-php.conf;
+        fastcgi_pass unix:/var/run/php/php7.4-fpm.sock;
+    }
+
+    location ~ /\.ht {
+        deny all;
+    }
+
+    location = /favicon.ico { log_not_found off; access_log off; }
+    location = /robots.txt { log_not_found off; access_log off; allow all; }
+    location ~* \.(css|gif|ico|jpeg|jpg|js|png)$ {
+        expires max;
+        log_not_found off;
+    }
+
+}
+
+server {
+    listen 80;
+    listen [::]:80;
+
+    server_name 192.168.50.2;
+
+    return 301 https://$server_name$request_uri;
```

Comment:
what this section is doing?

Reply:
Server_name points the DNS where it should go to connect to the server block.

Line 43: catches all http traffic and redirects it to https